.github/CONTRIBUTING.md

```diff
@@ -8,6 +8,16 @@ The following are certain Google Provider-specific things to be aware of when co
 
 We aim to make the Google Provider a good steward of Go practices. See https://github.com/golang/go/wiki/CodeReviewComments for common Go mistakes that you should attempt to avoid.
 
+## Autogenerated Resources
+
+We maintain 2 different versions of the google terraform provider; the [GA (General Audience) provider](https://github.com/terraform-providers/terraform-provider-google) and the [Beta provider](https://github.com/terraform-providers/terraform-provider-google). We are using code generation tool called Magic Modules that uses the same code base to generate both of these providers. Some Terraform resources are defined in `api.yaml` files [located in magic modules](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/products) and are fully generated, whereas some resources are hand written and located in [the provider/terraform/ folder in magic modules](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/provider/terraform/resources). Generated resources will have a prominent header at the top of the file identifying them.
+
+For more details on Magic Modules please visit [the readme](https://github.com/GoogleCloudPlatform/magic-modules). For feature requests or bugs regarding those resources, please continue to file [issues](https://github.com/terraform-providers/terraform-provider-google/issues) here. PRs changing those resources will not be accepted.
```

Comment:
Instead of asking them to file issues "here", let's be specific and say the "terraform-provider-google repositoriy's issue tracker".